locales-km-KH.xml

```diff
@@ -213,7 +213,7 @@
       <multiple>editors</multiple>
     </term>
     <term name="editorial-director">
-      <single/>
+      <single></single>
```

Comment:
Khmer was contributed by a former student who does ref manager training in Phnom Penh. I'll ping him so he can take a look. Meanwhile as-is is fine.